cobl2/datatables.py

```diff
@@ -33,6 +31,44 @@ def search(self, qs):
         return icontains(Meaning.name, qs)
 
 
+class CoblLanguages(Languages):
+    def get_options(self):
+        opts = super(Languages, self).get_options()
+        # default sort order on language name
+        opts['aaSorting'] = [[2, 'asc']]
+        return opts
+
+    def col_defs(self):
+        return [
+            IdCol(self, 'id', bSortable=False),
+            CoblCladeCol(self, 'Clade', model_col=Variety.clade),
+            CoblLgNameLinkCol(self, 'name'),
+            LinkToMapCol(self, 'm'),
+            Col(self, 'latitude',
+                sDescription='<small>The geographic latitude</small>'),
+            Col(self, 'longitude',
+                sDescription='<small>The geographic longitude</small>'),
+        ]
+
+
+class CoblLgNameLinkCol(LinkCol):
+    def format(self, item):
+        # add to link the historical info as superscript dagger if given
+        hist = '<sup title="historical">†</sup>' if item.historical else ''
+        obj = super(CoblLgNameLinkCol, self).format(item)
+        return '%s%s' % (obj, hist) if obj else ''
+
+
+class CoblCladeCol(Col):
+    def order(self):
+        return [Variety.clade, Language.name]
```

Comment:
Language names also appear in the DataTable anyway, and DataTables support sorting by multiple columns. So I'd remove this method entirely since the sorting by `Variety.clade` is already provided by the `model_col=Variety.clade` spec.

Reply:
but it should be sorted firstly by Variety.clade and in addition **secondly** by Language.name - where is it provided by `model_col=Variety.clade`?

Reply:
Defining sort criteria for more than one column may make sense as default - i.e. upon first rendering of the table. But after this, choosing sort options is up to the user - and a secondary criterion `Language.name` is in conflict with the ability of the user to order by language name themselves (e.g. choosing ascending order of clades but descending order of language names).

Reply:
Whatever is passed as `model_col` is used as return value for the base implementation of `order`.

Reply:
ah - ok - I forgot about SHIFT+click for adding an additional sort order of DataTables